diff -up discover-6.7.0/discover/discoversettings.kcfg.1~ discover-6.7.0/discover/discoversettings.kcfg
--- discover-6.7.0/discover/discoversettings.kcfg.1~	2026-06-11 11:31:10.000000000 +0200
+++ discover-6.7.0/discover/discoversettings.kcfg	2026-06-22 16:41:32.240179113 +0200
@@ -2,7 +2,7 @@
 <kcfg xmlns="http://www.kde.org/standards/kcfg/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.kde.org/standards/kcfg/1.0 http://www.kde.org/standards/kcfg/1.0/kcfg.xsd">
   <kcfgfile arg="true"/>
   <group name="DiscoverUI">
-    <entry name="appsListPageSorting" type="Int"><default>ResourcesProxyModel::RatingCountRole</default></entry>
+    <entry name="appsListPageSorting" type="Int"><default>ResourcesProxyModel::NameRole</default></entry>
     <entry name="installedPageSorting" type="Int"><default>ResourcesProxyModel::NameRole</default></entry>
     <entry name="installedPageCategorize" type="Bool"><default>true</default></entry>
   </group>
